packages/pubsub/src/snapshot.js

```diff
@@ -0,0 +1,195 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module pubsub/snapshot
+ */
+
+'use strict';
+
+var commonGrpc = require('@google-cloud/common-grpc');
+var is = require('is');
+var util = require('util');
+
+/**
+ * A Snapshot object will give you access to your Cloud Pub/Sub snapshot.
+ *
+ * Snapshots are sometimes retrieved when using various methods:
+ *
+ * - {module:pubsub#getSnapshots}
+ * - {module:pubsub#getSnapshotsStream}
+ * - {module:pubsub#snapshot}
+ *
+ * Snapshots may be created with:
+ *
+ * - {module:pubsub/subscription#createSnapshot}
+ *
+ * You can use snapshots to seek a subscription to a specific point in time.
+ *
+ * - {module:pubsub/subscription#seek}
+ *
+ * @alias module:pubsub/snapshot
+ * @constructor
+ *
+ * @example
+ * //-
+ * // From {module:pubsub#getSnapshots}
+ * //-
+ * pubsub.getSnapshots(function(err, snapshots) {
+ *   // snapshots is an array of Snapshot objects.
```

Comment:
I deliberately left those out because none of the examples use tick marks in pubsub. Maybe we should open a new issue to groom doc examples again.